src/components/DatasetOverviewTab/data/definitions.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+      "pdc": "Modified",
```

Comment:
Is there an API reason this variable is named "pdc"? "heading" would make the code much easier to understand.

Reply:
I used a csv to json convertor. This naming is based off of the column in the csv. I can updated it to heading, no problem.

Reply:
addressed
https://github.com/GetDKAN/cmsds-open-data-components/pull/239/commits/0e9cc4860083879df4674c696c79f2abebcfe381